### DISCUSSION TITLE
Defining an End to stop automation

### DISCUSSION THREAD

**Discussion by israelbarros** [AutomaApp/automa#70] [Ideas]
Hi there,

Automa is a really nice plugin and I enjoying to play with. I would like to suggest creating a block called STOP to be able to perform an automation once. 
My problem is when I use the trigger _when visit a website_, Automa run the automation every time I reload the website. 
It will be pleasant to stop it.

Thank you very much.

**Comment by Dayu0815** [comment]
The same situation as I encountered, I would like to ask if it is possible to adjust the trigger conditions of "on specific date" and "when visit a website". Only one-time triggers are accepted, and the workflow after the second trigger will not be started.

**Reply by Dayu0815** [reply]
Hello!
I'm sorry to bother you all the time. I would like to ask you about this problem. Is there any other way to solve it?

**Reply by Kholid060** [reply]
There's no other way to solve it right now
